modules/sb-about-ansible-service-broker.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-ansible-service-broker.adoc
+
+[id='sb-about-ansible-service-broker-{context}']
+= About the {asb-name}
```

Comment:
@shawn-hurley Can you take a look at this summary of the OpenShift Ansible broker (brought forward from 3.x) that I added in before getting in to installing it and see if it's good enough for 4.0 or if anything should be changed?

http://file.rdu.redhat.com/~ahoffer/2019/OSDOCS-176-update-installing-sb/applications/service_brokers/installing-ansible-service-broker.html

Reply:
Maybe a link to APB's repo so that users can look that up? 

Otherwise LGTM

Reply:
@shawn-hurley Are you talking about https://github.com/openshift/ansible-service-broker/ or something different?